pkg/callback/scheduler_callback.go

```diff
@@ -143,3 +145,48 @@ func (callback *AsyncRMCallback) ReSyncSchedulerCache(args *si.ReSyncSchedulerCa
 	}
 	return nil
 }
+
+// this callback implement scheduler plugin interface EventPlugin.
+func (callback *AsyncRMCallback) SendEvent(eventMessages []*si.EventMessage) error {
+	errors := make([]string, 0)
+	if len(eventMessages) > 0 {
+		log.Logger.Debug(fmt.Sprintf("processing %d events", len(eventMessages)))
+		for _, event := range eventMessages {
+			reason := event.Reason
+			msg := event.Message
+
+			switch event.Type {
+			case si.EventMessage_REQUEST:
+				taskAppID := event.ID
+				parts := strings.Split(taskAppID, ",")
```

Comment:
Instead of using a comma, can we use the naming convention like:
```
queueName/appID/taskID
```

this is more like a full-qualified name for an ID. If that makes sense.

Reply:
Modified accordingly.